dievai/pages/configuration.php

```diff
@@ -106,7 +101,7 @@
 					"0" => "{$lang['admin']['no']}", 
 					"3"	=> "{$lang['admin']['comments_off']}"
 				],
-				"selected" 	=> input(@$conf['kmomentarai_sveciams']),
+				"selected" 	=> input(@getSettingsValue('kmomentarai_sveciams')),
```

Comment:
```suggestion
				"selected" 	=> input(getSettingsValue('kmomentarai_sveciams')),
```
Galvoju ar dar reikalingi tie `@`